packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.ts

```diff
@@ -999,4 +1023,23 @@ export default class UserStorageController extends BaseController<
       );
     }
   }
+
+  async syncNetworks() {
```

Comment:
I dislike this controller now, it is a god controller.

TODO - Identity Team: Decouple the syncing features from UserStorageController. Create separate controllers for each sync features that then consume UserStorage through the messaging system. This way we can limit the permissions per sync feature instead of having really loose permissions on this controller!